### PR TITLE
ex-05-edgar

### PR DESCRIPTION
**Pull request by ewecke**

### CODE REVIEW

File: 04-PB-operators-arithmetic-pt2/04-ex.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
+"use strict"
```
Comment (martl) [inline 04-PB-operators-arithmetic-pt2/04-ex.js:1]: don't forget semicolons!

File: 06-PB-ternary-exercises/ex-06-edgar.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+let speedLimit = 50;
+console.log(speedCheck >= speedLimit ? 'you\'re going too fast!' : 'You\'re driving below the speed limit, Oma');
+
+//Q3
+let age = 40;
+console.log(age < 16 ? "serve butter beer" : "serve beer");
+
+//Q4
+let isStudent = true;
+console.log(isStudent = true ? "Ticket costs €5,00" : "Ticket costs €12,00");
```
Comment (Ungoot) [inline 06-PB-ternary-exercises/ex-06-edgar.js:17]: it should be `isStudent === true`

File: 06-PB-ternary-exercises/ex-06-edgar.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+//Q5
+let okMarie = true;
```
Comment (Ungoot) [inline 06-PB-ternary-exercises/ex-06-edgar.js:20]: We are checking if there is cake, so `let okMarie = "cake"` or `let okmarie = "bread"` would be better.

File: 06-PB-ternary-exercises/ex-06-edgar.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+console.log(okMarie = true ? "Let them eat cake" : "Oh, bother");
```
Comment (Ungoot) [inline 06-PB-ternary-exercises/ex-06-edgar.js:21]: single equals sign is not used for comparison, it's used for assignment. You should use 3 equals signs.

File: 07-PB-data-types-pt1/ex-07-edgar.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+//Q8
+console.log(`${name} is married: ${isMarried}`);
+
+//Q9
+let programmingLanguage = "JavaScript";
+let isFun = true;
+console.log(`${programmingLanguage} is Fun: ${isFun}`);
+
+//Q10
+console.log(`${programmingLanguage[0]+programmingLanguage[4]}`);
```
Comment (Ungoot) [inline 07-PB-data-types-pt1/ex-07-edgar.js:34]: This works but I would do it like this;

```
console.log(`${programmingLanguage[0]}${programmingLanguage[4]}`);

Comment (Ungoot) [reply 07-PB-data-types-pt1/ex-07-edgar.js:34]: You are using both concatenation and interpolation so choosing just one is better.

**Pull request closed without merging**